plugin-dev/Source/Sentry/Private/Android/Java/SentryBridgeJava.java

```diff
@@ -53,7 +57,8 @@ public void configure(SentryAndroidOptions options) {
 					options.setBeforeSend(new SentryOptions.BeforeSendCallback() {
 						@Override
 						public SentryEvent execute(SentryEvent event, Hint hint) {
-							return onBeforeSend(beforeSendHandler, event, hint);
+						checkForUnrealException(event);
```

Comment:
This is fairly messy. We modify an event based on tags the SDK should not set in the first place. And all of that happens in the java bridge code. Can we move this into a processor instead?

Reply:
Do you mean moving `checkForUnrealException` to C++? If so, we'll probably have to introduce another wrapper for Java `SentryException` which I was trying to avoid.